bluepyopt/deapext/CMA_SO.py

```diff
@@ -31,7 +31,7 @@
 
 from .stoppingCriteria import (
     MaxNGen,
-    Stagnation,
```

Comment:
What happened with Stagnation 'v1'?

Reply:
It still exists, but it was dependent on the evolution of median of the population, which was quite noisy, and not very reliant. Also it tended to trigger very late or even not at all, which made it not very good for our use cases. So I took the liberty to use the new stagnation criterion instead of the 'old' one.